README.md

```diff
@@ -100,5 +135,13 @@ Once in Vanilla JS, and once in jQuery, write a function that adds an event list
 
 Your Answer:
 ```js
-// write code here
+function feedback() {
+  alert("Great Job on Quiz 4!");
+}
+document.querySelector("submit-quiz").addEventListener("click", feedback)
```

Comment:
Awesome! Love the way you separated the callback function `feedback`. Just remember when doing Vanilla JS `document.querySelector(".submit-quiz") `you still need the` "." `to indicate a class